src/views/privacyPolicy.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import dwellinglyLogoMobile from "../assets/images/dwellingly_logo_white.png";
+import { Link } from "react-router-dom";
+
+
+
+const PrivacyPolicy = () => {
```

Comment:
No need for a change here, but JS does have short hand for an "implicit return" for arrow functions where instead of opening a method body, you just open parentheses for the body and it acts as a "return":

`const PrivacyPolicy = () => (
  <div>...</div>
)`